package.json

```
@@ -18,7 +18,8 @@
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
     "prepare": "husky install",
     "typecheck": "turbo run typecheck",
-    "storybook": "turbo run storybook"
+    "storybook": "turbo run storybook",
+    "storybook:build": "turbo run storybook"
```

Should't be the following here? 🤔 
```suggestion
    "storybook:build": "turbo run storybook:build"
```

---

yes... how is this already working? :D